data/data/bootstrap/files/usr/local/bin/wait-for-ha-api.sh

```diff
@@ -70,14 +71,21 @@ is_api_available() {
 	echo "$output"
 	return 1
     fi
+
+    threshold=2
+    if [ "$BOOTSTRAP_INPLACE" = true ]
+    then
+	threshold=1
+    fi
+    echo "Need at least $threshold available addresses for the default/kubernetes service"
```

Comment:
This will say

> Need at least 2

when in reality it needs 3

So maybe something like
```suggestion
    echo "Need at least $(($threshold+1)) available addresses for the default/kubernetes service"
```

Reply:
It needs 2 _durable_ instances. I think it might be best to determine this from the other side (that is, report the number of deployed instances managed by kube-apiserver operator instead of trying to subtract the number of temporary bootstrap instances from the service endpoint count).

What if we were to change the endpoints query to (untested) `oc --kubeconfig="$KUBECONFIG" get kubeapiservers cluster -o jsonpath='{range @.status.nodeStatuses[?(@.currentRevision>0)]}{.nodeName} {end}'`? The temporary bootstrap instances aren't managed by the operator, so this should only output nodes with operator-managed apiserver.

Reply:
I see. I think this is more clear and seems like it would resolve the agent issue we're seeing in testing. So let's see how the tests turn out!